src/api/spec/mailers/event_mailer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe EventMailer do
+  # Needed for X-OBS-URL
+  before { Configuration.any_instance.stubs(:obs_url).returns('https://build.example.com') }
+
+  context 'comment mail' do
+    let!(:receiver) { create(:confirmed_user) }
```

Comment:
But from example to example of the spec you can end up having the same project name if you don't ensure that